src/theme/variables/edgeDark.js

```diff
@@ -179,6 +179,11 @@ export const edgeDark: Theme = {
   settingsRowHeaderBackground: palette.edgeNavy,
   settingsRowSubHeader: palette.transparent,
 
+  // Wallet Loader
```

Comment:
Change to WalletProgressIcon. This should be reusable for the wallet list rows

Reply:
Fixed